d2l-icon.html

```diff
@@ -92,6 +103,7 @@
 
 			__onChange: function() {
 				this.__initialize();
+				this.updateStyles(); // need to re-calc iron-icon variables
```

Comment:
From the docs:
>>> Polymer's custom property shim evaluates and applies custom property values once at element creation time. In order to have an element (and its subtree) re- evaluate custom property values due to dynamic changes such as application of CSS classes, etc., call the updateStyles method on the element.

It's not that I'm setting the variable in JavaScript, but the `icon` property is getting changed after element creation, so the variables `d2l-icon-width` and `iron-icon-width` need to get re-evaluated for this subtree.